fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

```diff
@@ -476,4 +476,11 @@ object TestHelper {
             testBlock()
         }
     }
+
+    fun backgroundAndForegroundFenix() {
+        mDevice.pressRecentApps()
+        mDevice.findObject(UiSelector().resourceId("$packageName:id/container")).waitUntilGone(waitingTime)
+        mDevice.pressRecentApps()
+        mDevice.findObject(UiSelector().resourceId("$packageName:id/container")).waitForExists(waitingTime)
+    }
```

Comment:
I would split out this functionality into two functions.

`fun toggleAppToBackground()`
`fun toggleAppToForeground()`

Reply:
Done !